<?xml version="1.0"?>
<launch>
	<arg name="marker_size" default="10.0" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/s4n1/detection/image_rect_color" />
	<arg name="cam_info_topic" default="/s4n1/device/sensor0/head_camera/camera_info" />	
	<arg name="output_frame" default="s4n1/sensor0/head_camera_link" />

	<node name="app_test2" pkg="s4_operation" type="s4_operation_test2" respawn="false" output="screen">
		<remap from="objects" to="/s4n1/detection/objects" />
	</node>

</launch>


